### ISSUE TITLE
Disable encryption

### ISSUE THREAD

**Issue by amitgeron** [h2o/quicly#181]
Is there a way to disable encryption?

**Comment by janaiyengar** [comment]
QUIC is designed to be always encrypted and, as far as I know, there is no way to negotiate null ciphers with a peer. Why are you looking to disable encryption?

**Comment by amitgeron** [comment]
Hi Jana, thanks for your response! I want to inspect packets with Wireshark, for network traffic debugging. I'm attaching the captured packets using the cli application (8781ca5) against your server @ kazuhooku.com:4433, perhaps I'm missing something?
On the terminal the output I got is:
_./cli kazuhooku.com 4433
not found
packets-received: 7, packets-sent: 6, packets-lost: 0, ack-received: 2, bytes-received: 2754, bytes-sent: 2776, srtt: 325_

[QUIC_kazuhooku.com_4433.zip](https://github.com/h2o/quicly/files/3226161/QUIC_kazuhooku.com_4433.zip)

**Comment by janaiyengar** [comment]
Yes, wireshark will be of quite limited value because the packets are encrypted. You might find event logging useful. You can log using the -e command line option, which gives you all the events at the client including packet sent and received events: `./cli -e logfile.json kazuhooku.com 4433`

If you'd like to get a packet trace from the event log, you can use quictrace (https://github.com/google/quic-trace). There's also a convenience script, `plot.sh`, in the quicly repo that will do the translation from a quicly log and generate a plot using quic-trace.

**Comment by jaysec12345** [comment]
Note: In Issue #99 decryption in wireshark by using the temporary session keys was mentioned

**Comment by hrj** [comment]
> QUIC is designed to be always encrypted and, as far as I know, there is no way to negotiate null ciphers with a peer. Why are you looking to disable encryption?

One use-case for no encryption: performance constrained embedded systems within a trusted network.
